lib/src/presentation/line_numbers_wrapper.dart

```diff
@@ -0,0 +1,54 @@
+import 'package:flutter/material.dart';
+import 'package:kode_view/src/presentation/styles/text_styles.dart';
+
+class LineNumbersWrapper extends StatelessWidget {
+  const LineNumbersWrapper({
+    required this.enableLineNumbers,
+    required this.linesNumber,
+    required this.child,
+    this.textStyle,
+    this.lineNumberBackgroundColor,
+    this.lineNumberColor,
+    super.key,
+  });
+  final bool enableLineNumbers;
+  final int linesNumber;
+  final Color? lineNumberBackgroundColor;
+  final Color? lineNumberColor;
+  final TextStyle? textStyle;
+  final Widget child;
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      children: [
+        if (enableLineNumbers) ...[
+          Container(
+            color: lineNumberBackgroundColor,
+            width: 40,
```

Comment:
I think we should have default spacings, but also enable customization. What about line 11111? This will overflow?